usage.py

```diff
@@ -386,7 +386,15 @@
 
         html.Div([
                     html.P([html.Strong('Sample for Stepper')]),
-                    sd_material_ui.Stepper(id='stepper', style={'background': 'inherit'}),
+                    sd_material_ui.Stepper(id='stepper',
+                                           style={'background': 'inherit'},
+                                           # nextButtonStyle={'display': 'None'},
```

Comment:
Maybe remove these comments?

Reply:
since this is merging commits directly from the master repo, probably a good idea to avoid changing stuff